R/utils.R

```diff
@@ -34,8 +34,8 @@
 #' - When the length of `size` is three: the plot points size are dynamically adjusted based on
 #' vector of `value`, `min`, and `max`.
 #' @param decorators `r lifecycle::badge("experimental")`
-#' (`list` of `teal_transform_module`, named `list` of `teal_transform_module` or `NULL`) optional,
-#' if not `NULL`, decorator for tables or plots included in the module.
+#' (`list` of `teal_transform_module`, named `list` of `teal_transform_module`) optional,
+#' decorator for tables or plots included in the module.
```

Comment:
Suggestion to make it clear the distinction between transformers and decorators

```suggestion
#' decorator for tables or plots included in the module output reported.
```